app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-<h1>Details of posts for a given user</h1>
\ No newline at end of file
+<a href="<%= users_path() %>">
+  <button class="custom-back-btn">⬅</button>
+</a>
+<ul>
+  <a href="<%= user_path(@user) %>">
+    <div class="custom-user-box">
+      <%= render partial: 'user_helper/user_photo', locals: { user: @user } %>
+      <%= render partial: 'user_helper/user_details', locals: { user: @user } %>
+    </div>
+  </a>
+  <% @posts.each do |post|%>
+    <a href="<%= user_post_path(@user, post) %>">
+      <div class="custom-user-post">
+        <div class="custom-post">
+          <h3>Post <%= raw "#" + post.id.to_s %></h3>
+          <p><%= post.text %></p>
+          <div class='custom-post-interactions'>
+            <span>Comments: <%= post.comments_counter %>,</span>
+            <span>Likes: <%= post.likes_counter %></span>
+          </div>
+        </div>
+        <h3>Comments:</h3>
+        <% post.recent_comments.each do |comment, index| %>
+          <div class="custom-post-comments">
+            <span><%= comment.author.name %>:</span>
```

Comment:
- [ ] there is an error when I try to access your post index due to some misspelling,` comments_counter` should be `commentsCounter`,` likes_counter` should be l`ikesCounter` and also it can not access `comment.author.name` kindly fix these issues
![Screenshot from 2023-10-05 22-43-42](https://github.com/leloufadel/my_blog_app/assets/66097371/ab68d319-ffbf-476c-a868-ed8c766a7e0e)